packages/terra-button-group/src/ButtonGroupButton.jsx

```diff
@@ -128,12 +135,28 @@ class ButtonGroupButton extends React.Component {
       customProps.className,
     );
 
+    if (icon) {
+      return (
+        <IconButton
+          {...customProps}
+          icon={icon}
+          iconType={iconType}
+          isDisabled={isDisabled}
+          isIconOnly={icon != null}
```

Comment:
isIconOnly should always be true here.

Reply:
[updated](https://github.com/cerner/terra-core/pull/3640/commits/c402015452ed0e585360b79f61e22fdcdf7c6f01)